config/200-config.yaml

```diff
@@ -44,3 +44,8 @@ data:
     # in the context of services should be logged. Readiness
     # probes etc. must be configured via the bootstrap config.
     enable-service-access-logging: "true"
+
+    # Specifies whether to use proxy-protocol in ordrer to to safely
+    # transport connection information such as a client's address
+    # across multiple layers of TCP proxies.
```

Comment:
Could you mention that this is an experimental/alpha feature just in case?

https://github.com/knative/serving/blob/77a0443c52fba683abd88a76e645244cb38e30ac/config/core/configmaps/deployment.yaml#L101